web-js/src/components/WelcomePage.js

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+import * as Realm from "realm-web";
+import {
+  Container,
+  TextField,
+  Button,
+  IconButton,
+  Card,
+  Typography,
+  InputAdornment,
+} from "@material-ui/core";
+import { useRealmApp } from "./RealmApp";
+import { MoreInfoTemplateAndDocs } from './MoreInfo'
+
+import Visibility from "@material-ui/icons/Visibility";
+import VisibilityOff from "@material-ui/icons/VisibilityOff";
+
+export function WelcomePage() {
+  const realmApp = useRealmApp();
+  // Track whether the user is logging in or signing up for a new account
+  const [isSignup, setIsSignup] = React.useState(false);
+  const toggleIsSignup = () => {
+    clearErrors();
+    setIsSignup((s) => !s);
+  };
+  // Authentication errors
+  const noErrors = {
+    email: null,
+    password: null,
+  };
+  const [error, setError] = React.useState(noErrors);
+  const clearErrors = () => setError(noErrors);
+  // Manage password visibility
+  const [showPassword, setShowPassword] = React.useState(false);
+  const toggleShowPassword = () => setShowPassword((s) => !s);
+
+  const onFormSubmit = async ({ email, password }) => {
+    clearErrors();
+    try {
+      if (isSignup) {
+        await realmApp.emailPasswordAuth.registerUser(email, password);
```

Comment:
When a user creates an account, they don't see anything in the UI that the account has been successfully created. This lead me to click the button twice and get an error saying the account has already been created. 

Can you do one of the following:
* Add some UI message saying 'your account has been successfully created' once it has been created.
or 
* Log the user in after the user has registered

Reply:
Good catch! Updated to auto-login on successful signup.